contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -264,6 +271,23 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, IERC677Rece
     return s_subscriptions[subscriptionId];
   }
 
+  // @inheritdoc IFunctionsSubscriptions
```

Comment:
Notice that this is disabled. To get this in we will need to shave 643 bytes off of FunctionsRouter.sol from somewhere